liberty-maven-plugin/src/it/tests/config-packagefile-all-it/src/test/resources/server.xml

```diff
@@ -0,0 +1,6 @@
+<server description="default server">
+    <featureManager>    
+        <feature>osgiBundle-1.0</feature>
+        <feature>jsp-2.3</feature>
```

Comment:
you may not need any features.  Try taking them out, so we are wondering later why we needed osgiBundle or jsp.